src/routes/NavigationWrapper.tsx

```diff
@@ -1,37 +1,67 @@
+import PersonIcon from "@material-ui/icons/Person";
+import RoomIcon from "@material-ui/icons/Room";
+import ScheduleIcon from "@material-ui/icons/Schedule";
 import React from "react";
-import { Clock, MapPin, User } from "react-feather";
 import { Route, Switch } from "react-router-dom";
 import { NavigationBar } from "~/components/NavigationBar";
 import { Helpers, Jobs, Profile } from "~/pages/private";
 import { Login, Map } from "~/pages/public";
-
-const helperTabs = [
-  { route: "/map", icon: <MapPin />, value: "nearby", label: "In der Nähe" },
-  { route: "/jobs", icon: <Clock />, value: "jobs", label: "Jobs" },
-  { route: "/profile", icon: <User />, value: "profile", label: "Profil" },
-  { route: "/login", icon: <User />, value: "login", label: "Login" }
-];
+import { useTypedSelector } from "~/reducers";
+import { userIsAuthenticated } from "~/routes/auth";
 
 const providerTabs = [
   {
     route: "/helpers",
-    icon: <Clock />,
+    icon: <ScheduleIcon />,
     value: "helpers",
     label: "Meine Helfer"
   },
-  { route: "/profile", icon: <User />, value: "profile", label: "Meine Daten" }
+  {
+    route: "/profile",
+    icon: <PersonIcon />,
+    value: "profile",
+    label: "Meine Daten"
+  }
 ];
 
 const NavigationWrapper = () => {
+  const user = useTypedSelector((state) => state.get("user"));
   const helper = true;
-  // FIXME Use user data from store
+  let tabs;
+
+  if (helper) {
+    tabs = [
+      {
+        route: "/map",
+        icon: <RoomIcon />,
+        value: "nearby",
+        label: "In der Nähe"
+      },
+      { route: "/jobs", icon: <ScheduleIcon />, value: "jobs", label: "Jobs" },
+      user.get("login")
+        ? {
+            route: "/profile",
+            icon: <PersonIcon />,
+            value: "profile",
+            label: "Profil"
+          }
+        : {
+            route: "/login",
+            icon: <PersonIcon />,
+            value: "login",
+            label: "Login"
+          }
+    ];
+  } else {
+    tabs = providerTabs;
+  }
```

Comment:
Was hälst Du davon, diesen Teil der Lesbarkeit wegen in z. B. eine `useNavigationTabs` hook o. ä. auszulagern?

Reply:
Finde ich aktuell noch nicht zu meiner Zufriedenheit gelöst.
So würde das Erstellen der Tabs in ne Hook ausgelagert werden? Warum Hook und nicht ne einfache Funktion? Weil das ja im Endeffekt auch nur hier genutzt wird und nicht mehrfach oder?

Reply:
Grundsätzlich reicht eine einfache Funktion natürlich auch aus. Ich tendiere nur dazu, sie als Hooks zu bezeichnen, damit ihnen auch eine gewisse Konvention einhergeht (Name, Verwendung, Platzierung innerhalb der Komponente, ...).

Stimme dir daber dabei zu, dass wir dafür noch eine bessere Lösung finden können. Da es aber aktuell seinen Zweck erfüllt, würde ich es vorerst so lassen, sofern Du es nicht in eine Funktion bzw. eine Hook auslagern möchtest.